.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,67 @@
+name: "\U0001F41E Report a problem"
+description: "Report an issue with ESLint or rules bundled with ESLint"
+title: "Bug: (fill in)"
+labels:
+  - bug
+  - "repro:needed"
+body:
+- type: markdown
+  attributes:
+    value: ESLint adheres to the [Open JS Foundation Code of Conduct](https://eslint.org/conduct). By opening an issue, you agree to abide by the code of conduct.
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      Please tell us about how you're running ESLint (Run `npx eslint --env-info`.)
+    value: |
+        Node version: 
+        npm version: 
+        Local ESLint version: 
+        Global ESLint version: 
+        Operating System:
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: What parser are you using?
+    description: |
+      Please keep in mind that some problems are parser-specific.
+    options:
+      - "Default (Espree)"
+      - "@typescript-eslint/parser"
+      - "@babel/eslint-parser"
+      - "vue-eslint-parser"
+      - "@angular-eslint/template-parser"
+      - Other
```

Comment:
Do we care about versions for these parsers, and should there be an optional text input to specify an "Other" parser?

Reply:
We aren’t collecting that data currently, so I’m guessing it’s not important.

I can’t find a way to do an optional text field, and given how rare this is likely to be, I think we roll with this and see what happens.